tests/contrib/openai/conftest.py

```diff
@@ -53,6 +53,11 @@ def openai_organization():
     return None
 
 
+@pytest.fixture
+def oai_with_test_agent_backend(openai):
+    return openai.OpenAI(base_url="http://127.0.0.1:9126/vcr/openai")
```

Comment:
I'm not certain, but it seems like this would be improved by using whatever config value `DD_AGENT_URL` adjusts